source/sphinx_extensions/command_block/extension.py

```diff
@@ -40,17 +63,34 @@ class CommandBlockDirective(docutils.parsers.rst.Directive):
     has_content = True
 
     option_spec = {
-        'no-exec': docutils.parsers.rst.directives.flag
+        'no-exec': docutils.parsers.rst.directives.flag,
+        'url': docutils.parsers.rst.directives.unchanged_required,
+        'rename': docutils.parsers.rst.directives.unchanged_required,
```

Comment:
What do you think of using `saveas` instead of `rename`? It's a little clearer to me, but I also don't have strong feelings on this option's name.